Makefile

```diff
@@ -33,19 +41,8 @@ BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
-# IMAGE_TAG_BASE defines the docker.io namespace and part of the image name for remote images.
-# This variable is used to construct full image tags for bundle and catalog images.
-#
-# For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
-# grafana.com/tempo-operator-bundle:$VERSION and grafana.com/tempo-operator-catalog:$VERSION.
-IMAGE_TAG_BASE ?= grafana.com/tempo-operator
```

Comment:
`CATALOG_IMG` still references this variable - can you update `CATALOG_IMG` to use `IMG_PREFIX` etc., like the `BUNDLE_IMG`?